ansible/roles/ocp4-workload-integreatly/sample_vars/ocp4_workload_integreatly_vars.yml

```diff
@@ -0,0 +1,5 @@
+---
+# override ocp workloads to include prerequisite roles
+ocp_workloads:
+  - ocp4-workload-integreatly-minio
+  - ocp4-workload-integreatly
```

Comment:
Wasn't sure where to put this, but it's useful for local dev and it's the order we'll need the roles run in in the actual workshop on RHPDS.